src/app.ts

```diff
@@ -1,6 +1,89 @@
+type ResultData = {
+  id: string;
+  gender: string;
+  age: number;
+  row: number;
+};
+
+type PagingData = {
+  next?: string;
+  previous?: string;
+};
+
+let previousBtn: HTMLElement | any = document.querySelector("button[data-prevbtn]");
+let nextBtn: HTMLElement | any = document.querySelector("button[data-nextbtn]");
+let pageView: HTMLElement | any = document.querySelector("label[data-pageview]");
```

Comment:
Codacy has a fix for the issue: [Replace `"label[data-pageview]"` with `⏎··"label[data-pageview]"⏎`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
let pageView: HTMLElement | any = document.querySelector(
```